merge_sources/kubernetes/cloud_native_postgresql/interactive_demo.mdx

```diff
@@ -213,11 +213,11 @@ status:
   readService: cluster-example-r
   readyInstances: 3
   secretsResourceVersion:
-    applicationSecretVersion: "1479"
-    caSecretVersion: "1475"
-    replicationSecretVersion: "1477"
-    serverSecretVersion: "1476"
-    superuserSecretVersion: "1478"
+    applicationSecretVersion: "612"
```

Comment:
Don't suppose it matters what these numbers are? Looks like they are just getting incremented as people run the demo or somesuch?

Reply:
https://github.com/EnterpriseDB/docs/blob/6a84ab2007c2240e02d6d924214e547a9ec4e523/advocacy_docs/kubernetes/cloud_native_postgresql/api_reference.mdx#secretsresourceversion